_posts/2023-03-09-solidity-developer-survey-2022-results.md

```diff
@@ -0,0 +1,452 @@
+---
+layout: post
+published: true
+title: 'Solidity Developer Survey 2022 Results'
+date: '2023-03-09'
+author: Franziska Heintel
+category: Announcements
+---
+
+In this post, we will be summarizing and analyzing the results of the [2022 Solidity Developer Survey](https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/).
+
+First of all, a big thank you to everybody who took the time and participated and to everybody who helped us spread the word about it!
+
+This year, we received a smashing 1401 responses. That is more than a 3x in responses compared to the previous survey and we couldn't be happier with the turnout.
+
+Your input is invaluable to us and plays a crucial role in helping to continuously improve the Solidity developer experience as a whole.
+
+Before we get started, a few useful links:
+* In the spirit of open source, you can find all raw data of the survey results [here](https://docs.google.com/spreadsheets/d/1A5iF3aKhFv9wTTJ10ko_uxgoflc8oEpVe3gqAAWoC2w/edit?usp=sharing) and all graphs [here](https://docs.google.com/presentation/d/1xH5pGZ6rrAP_jzRQobf0Mn1XYquyg8bD17DBQyrErMo/edit?usp=sharing).
+* Since this is already our third time conducting a yearly survey, it may be interesting for you to compare the outcome to the previous surveys. The results from the 2021 developer survey are available [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/) and from 2020 [here](https://blog.soliditylang.org/2021/01/26/solidity-developer-survey-2020-results/).
+
+Without further ado, let’s dig into the 2022 results!
```

Comment:
```suggestion
Before we get started, here are a few useful links:
* In the spirit of open source, you can find all raw data of the survey results [here](https://docs.google.com/spreadsheets/d/1A5iF3aKhFv9wTTJ10ko_uxgoflc8oEpVe3gqAAWoC2w/edit?usp=sharing) and all graphs [here](https://docs.google.com/presentation/d/1xH5pGZ6rrAP_jzRQobf0Mn1XYquyg8bD17DBQyrErMo/edit?usp=sharing).
* Since this is already our third time conducting a yearly survey, it may be interesting for you to compare the outcome to the previous surveys. The results from the 2021 developer survey are available [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/) and from 2020 [here](https://blog.soliditylang.org/2021/01/26/solidity-developer-survey-2020-results/).

Without further ado, let’s dig into the 2022 results!
```